hi.py

```diff
@@ -0,0 +1,8 @@
+def hello():
+  return "Hello User" 
+
+    
+    
+    
+if name == "__name___"
```

Comment:
[![Panto AI](https://raw.githubusercontent.com/pantomaxdotco/assets/refs/heads/main/pr-comment-branding-50.png)](https://www.pantomax.co)
The condition should be `if __name__ == "__main__":` instead of `if name == "__name___"`.